server/v2/server.go

```diff
@@ -4,20 +4,24 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"path/filepath"
 
-	"github.com/pelletier/go-toml/v2"
+	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
 	"golang.org/x/sync/errgroup"
 
+	"cosmossdk.io/core/transaction"
 	"cosmossdk.io/log"
+	"github.com/pelletier/go-toml/v2"
 )
 
 // ServerModule is a server module that can be started and stopped.
```

Comment:
Can we rename this to something like `ServerComponent` everywhere? To not get too many things called modules.